website/docs/d/nat_gateway.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "aws"
+page_title: "AWS: aws_nat_gateway"
+sidebar_current: "docs-aws-datasource-nat_gateway"
+description: |-
+    Provides details about a specific Nat Gateway
+---
+
+# aws\_nat\_gateway
+
+`aws_nat_gateway` provides details about a specific Nat Gateway.
+
+## Example Usage
+
+```hcl
+variable "subnet_id" {}
+
+data "aws_nat_gateway" "default" {
+
+  subnet_id = "${var.subnet_id}"
+  
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available
+Nat Gateway in the current region. The given filters must match exactly one
+Nat Gateway whose data will be exported as attributes.
+
+* `nat_gateway_id` - (Optional) The id of the specific Nat Gateway to retrieve.
```

Comment:
This needs to be `id` but I'm waiting to see if any other bugs exist first.

Reply:
You can name it `id` indeed